gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -153,8 +153,12 @@
   private static final String LIST_OBJECT_FIELDS_FORMAT = "items(%s),prefixes,nextPageToken";
 
   // To track the object statistics
-  private HashMap<GoogleCloudStorageStatistics, AtomicLong> objectStatistics =
-      new HashMap<GoogleCloudStorageStatistics, AtomicLong>();
+  private final ConcurrentHashMap<GoogleCloudStorageStatistics, AtomicLong> objectStatistics =
+      new ConcurrentHashMap<GoogleCloudStorageStatistics, AtomicLong>();
+
+  // To track the http statistics
+  private static final ConcurrentHashMap<GoogleCloudStorageStatistics, AtomicLong> httpStatistics =
```

Comment:
Please remove static

Reply:
Can we please fix this? We have missed this in the last 3 iterations

Reply:
Updated